tests/DCPS/Compiler/anonymous_types/anonymous_types.mpc

```diff
@@ -0,0 +1,7 @@
+project: dcps {
+  idlflags += -as
+  dcps_ts_flags += -as
```

Comment:
This shouldn't be needed.  Remove it, and then if the test breaks make any changes needed in the opendds_idl source code.

Reply:
Same as above.